packages/ckeditor5-list/src/list/converters.ts

```diff
@@ -81,6 +81,7 @@ export function listItemUpcastConverter(): GetCallback<UpcastElementEvent> {
 
 		const listItemId = ListItemUid.next();
 		const listIndent = getIndent( data.viewItem );
+		const todoListChecked = isTodoListChecked( items );
```

Comment:
I'd suggest fixing the to-do list converter instead of modifying the generic list converter.

https://github.com/ckeditor/ckeditor5/blob/897019cfb90f76bc4716332dfaf41a6f883d92f0/packages/ckeditor5-list/src/todolist/todolistediting.ts#L398-L424

Reply:
@niegowski I moved fix to to-do list converter in this [commit](https://github.com/ckeditor/ckeditor5/pull/16785/commits/3d2aed708ede0c5880b430f05d70861c09055cd6). Can you take a look?